addons/mail/static/src/js/activity.js

```diff
@@ -349,6 +350,35 @@ var Activity = AbstractActivityField.extend({
             }).popover('show');
         }
     },
+    _onClickMailTemplate: function (event) {
```

Comment:
Maybe there are some JS guidelines to improve here (I don't know them all, but in short)
 - missing docstring
 - I think event should be ev (not sure)
 - some spacing issues